pdf/core/utils.go

```diff
@@ -38,6 +38,18 @@ func getUniDocVersion() string {
 	return common.Version
 }
 
+// GetObjectNums returns the object numbers of the PDF objects in the file
+func (this *PdfParser) GetObjectNums() (map[int]int, error) {
```

Comment:
OK.  I will add such a function and verify that it works with pdf_all_objects.go